sim/motorIoc/configure/RELEASE

```diff
@@ -0,0 +1,17 @@
+SUPPORT= /home/bar/EPICS/support
```

Comment:
Home directory path in repo ⛔ 

The way around this would be to create `RELEASE.local` locally on the development machine and override the `SUPPORT` macro there. That file should then be gitignored. 

However, this is also bit much work/instructions to build the system later. So a gold-plated solution would be to make a `configure.sh` script to run before building everything - and that script would simply generate `RELEASE.local` files for all the IOCs, probably with just a single line: `SUPPORT=/path/to/support/dir`